DataFormats/Detectors/TRD/include/DataFormatsTRD/PID.h

```diff
@@ -30,44 +31,69 @@ enum class PIDPolicy : unsigned int {
   LQ1D = 0, ///< 1-Dimensional Likelihood model
   LQ3D,     ///< 3-Dimensional Likelihood model
 
+#ifdef TRDPID_WITH_ONNX
   // ML models
-  XGB, ///< XGBOOST
+  XGB,        ///< XGBOOST
+  PY,         ///< Pytorch
+  Test = XGB, ///< Load object for testing
+#endif
 
   // Do not add anything after this!
   NMODELS,         ///< Count of all models
-  Test,            ///< Load object for testing
   Dummy,           ///< Dummy object outputting -1.f
   DEFAULT = Dummy, ///< The default option
 };
 
+inline std::ostream& operator<<(std::ostream& os, const PIDPolicy& policy)
+{
+  std::string name;
+  switch (policy) {
+    case PIDPolicy::LQ1D:
+      name = "LQ1D";
+      break;
+    case PIDPolicy::LQ3D:
+      name = "LQ3D";
+      break;
+#ifdef TRDPID_WITH_ONNX
```

Comment:
In case O2 is compiled without ONNX and then you try to put the policy `XGB` for example then without an error the default dummy option would be taken instead, right? I think it would be better to have a dummy policy, but to throw an error by default if the selected policy does not exist